gflownet/gflownet.py

```diff
@@ -475,12 +460,27 @@ def sample_batch(
                 dict_replay = pickle.load(f)
                 n_replay = min(n_replay, len(dict_replay["x"]))
                 envs = [self.env.copy().reset(idx) for idx in range(n_replay)]
-                # TODO: implement other sampling options besides permutation
                 if n_replay > 0:
                     x_replay = [x for x in dict_replay["x"].values()]
```

Comment:
Convert to list through `list()` instead of through iteration (a bit faster and more readable). This was my code, admittedly.
```suggestion
                    x_replay = list(dict_replay["x"].values())
```